### PR TITLE
Feature/comin nfcens

### PR DESCRIPTION
**Pull request by SamiraArdani-NOAA**
## Pull Request Testing ##

- [ ] Describe testing already performed for this Pull Request:</br>
COMINxxx variable were updated to EVSINxxx.
- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
Driver scripts are located here:  
prep: /lfs/h2/emc/vpppg/noscrub/samira.ardani/EVS/dev/drivers/scripts/prep/nfcens/jevs_nfcens_wave_grid2obs_prep.sh
stats: /lfs/h2/emc/vpppg/noscrub/samira.ardani/EVS/dev/drivers/scripts/stats/nfcens/jevs_nfcens_wave_grid2obs_stats.sh
plots: /lfs/h2/emc/vpppg/noscrub/samira.ardani/EVS/dev/drivers/scripts/prep/nfcens/jevs_nfcens_wave_grid2obs_plots.sh

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes]**

- [ ] Do these updates/additions include sufficient testing updates? **[Yes]**

- [ ] Please complete this pull request review by **[10/20/2023]**.</br>

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

File: jobs/JEVS_NFCENS_WAVE_GRID2OBS_PREP [hunk 1]
```diff
@@ -57,7 +57,7 @@ export VERIF_CASE=${VERIF_CASE:-grid2obs}
 ####################################
 export COMIN=${COMIN:-$(compath.py ${envir}/com/${NET}/${evs_ver})}
 export COMINmodel=${COMINmodel:-$(compath.py ${envir}/com/${RUN}_${MODELNAME}/${nfcens_ver})}
```
Comment (ShelleyMelchior-NOAA) [inline jobs/JEVS_NFCENS_WAVE_GRID2OBS_PREP]: Let's use the specific model name. So please change COMINmodel to COMINnfcens.

File: jobs/JEVS_NFCENS_WAVE_GRID2OBS_STATS [hunk 2]
```diff
@@ -58,7 +58,7 @@ export VERIF_CASE=${VERIF_CASE:-grid2obs}
 ####################################
 export COMIN=${COMIN:-$(compath.py ${envir}/com/${NET}/${evs_ver})}
 export COMINmodel=${COMINmodel:-$(compath.py ${envir}/com/${RUN}_${MODELNAME}/${nfcens_ver})}
```
Comment (ShelleyMelchior-NOAA) [inline jobs/JEVS_NFCENS_WAVE_GRID2OBS_STATS]: Same thing, let's use the specific model name. Change COMINmodel to COMINnfcens.

Comment (SamiraArdani-NOAA) [reply jobs/JEVS_NFCENS_WAVE_GRID2OBS_STATS]: @ShelleyMelchior-NOAA 
I applied the changes.

Comment (ShelleyMelchior-NOAA) [reply jobs/JEVS_NFCENS_WAVE_GRID2OBS_STATS]: > @ShelleyMelchior-NOAA I applied the changes.

Great -- can you push them? I don't seen them in the PR yet.

**Comment by ShelleyMelchior-NOAA** [comment]
Hi @SamiraArdani-NOAA , I ran prep. Please review.
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/prep/nfcens/wave.20231015`
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_nfcens_wave_grid2obs_prep.o106494410`

**Comment by SamiraArdani-NOAA** [comment]
Hi @ShelleyMelchior-NOAA,
I checked "prep" files for nfcens. They look good.

**Comment by ShelleyMelchior-NOAA** [comment]
The stats processing has completed. Please review.
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/nfcens/`
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_nfcens_wave_grid2obs_stats.o106508075`

**Comment by SamiraArdani-NOAA** [comment]
"stats" files for nfcens were reviewed. They look good.

**Comment by ShelleyMelchior-NOAA** [comment]
plots processing has completed. Please review.
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/plots/nfcens/wave.20231015`
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_nfcens_grid2obs_plots.o106509055`

**Comment by SamiraArdani-NOAA** [comment]
"plots" were reviewed. Everything looks good.

**Comment by ShelleyMelchior-NOAA** [comment]
Excellent! I will do the formal code review and then approve this PR for merge.

File: jobs/JEVS_NFCENS_WAVE_GRID2OBS_PREP [hunk 3]
```diff
@@ -56,8 +56,8 @@ export VERIF_CASE=${VERIF_CASE:-grid2obs}
 # Define COMIN/COMOUT variables     
 ####################################
 export COMIN=${COMIN:-$(compath.py ${envir}/com/${NET}/${evs_ver})}
-export COMINmodel=${COMINmodel:-$(compath.py ${envir}/com/${RUN}_${MODELNAME}/${nfcens_ver})} 
-export COMINgdas=${COMINobs:-$(compath.py ${envir}/com/obsproc/${obsproc_ver}/gdas)}
+export COMINnfcens=${COMINnfcens:-$(compath.py ${envir}/com/${RUN}_${MODELNAME}/${nfcens_ver})} 
+export COMINobsproc=${COMINobs:-$(compath.py ${envir}/com/obsproc/${obsproc_ver}/gdas)}
```
Comment (AliciaBentley-NOAA) [inline jobs/JEVS_NFCENS_WAVE_GRID2OBS_PREP]: @SamiraArdani-NOAA Can you please change COMINobs to COMINobsproc in this line?

export COMINobsproc=${**COMINobsproc**:-$(compath.py ${envir}/com/obsproc/${obsproc_ver}/gdas)}

**Review by ShelleyMelchior-NOAA — APPROVED** [review]
Testing yields expected results. This PR is approved for merge.

**Comment by SamiraArdani-NOAA** [comment]
Thank you.

**Pull request merged**